app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create_commit :update_likes_counter
+
+  def update_likes_counter
+    post.update(likes_count: post.likes.count)
+  end
```

Comment:
[OPTIONAL]
- [ ] Same for the `update_likes_counter` as pointed out in the previous comment :point_up_2: Kindly make it private to follow the best practices